pkg/kernel/tools/nshandle/ns_handle.go

```diff
@@ -49,6 +52,26 @@ func FromURL(urlString string) (handle netns.NsHandle, err error) {
 	return handle, nil
 }
 
+// ToNetlinkHandle - mechanism to netlink.Handle for the NetNS specified in mechanism
+func ToNetlinkHandle(urlString string) (*netlink.Handle, error) {
```

Comment:
I am not sure that it is somehow related to the `nshandle`. Probably it would be better to create separate package `netlinkhandle` with `FromURL()` method.

Reply:
ok, just moved these 2 methods into link.go module which looked to be more relevant, is that fine ?